### PR TITLE
Les 67

### PR DESCRIPTION
**Pull request by IgorTanov**

### CODE REVIEW

File: src/services/services.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const getData = async () => {
+  await fetch("https://jsonplaceholder.typicode.com/posts/1")
```
Comment (Terapool) [inline src/services/services.tsx:2]: I am not sure that here asyn / await is needed as you already use fetch().then 
Choose 1 variant.

Example: https://dev.to/masteringjs/using-then-vs-async-await-in-javascript-2pma#:~:text=Async%2Fawait%20and%20then(),callback%20until%20the%20promise%20settles.

File: src/App.test.tsx [hunk 2]
```diff
@@ -1,8 +1,8 @@
-import React from 'react';
-import { render, screen } from '@testing-library/react';
-import App from './App';
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import App from "./App";
 
-test('renders learn react link', () => {
+test("renders learn react link", () => {
   render(<App />);
```
Comment (Terapool) [inline src/App.test.tsx:6]: Remove this file as you  changed App.tsx and broke the test

**Review by Terapool — COMMENTED** [review]
The Les-67 should relate to the linter changes. I see other logic in this Pull request. E.g. Main, Header components. 

Please leave only files and changes that concerns only linter

File: src/services/services.tsx (hunk 1 above)
Comment (Evgeniy121212) [reply src/services/services.tsx:2]: Please, use `async/await` variant - it's current modern style

**Pull request closed without merging**